script/Ciel/Day 14/14_01B_03_CIEL14B_4_1B.txt

```diff
@@ -186,92 +200,94 @@ Your body isn't the only thing that's tough.
 [sha:e01972ce109d5e072fbf58d681ccb26e897f6e32]{
 -- Page 31, Offset 40337.
 -- まだ理性があるなんて、どうなってるのシエル？”」
-How are you still holding on to reason after all that, Ciel?"
+Just how have you managed to stay sane through all this, Ciel?"
 }
 [sha:ae125078c1b2be40e928421b29ff9d8f7bbbfa18]{
 -- Page 32, Offset 40338.
 -- 指先で必死にもがく人間に語りかける。
-She speaks to the human who squirms desperately at her fingertips.
+She addresses the human desperately squirming between her fingertips.
 }
 [sha:1545b5cbf5523157774f5514ebffbb007df61089]{
 -- Page 32, Offset 40339.
 -- この人間とはもう幾度となく意思を交わしあった。
-She had already clashed wills with this human on countless occasions.
+She has already butted heads with this human on countless occasions.
 }
 [sha:412c13ffd48236f30ee814aebb9fcc7d2d792852]{
 -- Page 32, Offset 40340.
 -- 八つ裂きにしても飽き足らなかった女だが、今は奇妙な連帯感すら抱きつつある事が、彼女を余計に残酷にさせていた。
-Even though she never grew bored of tearing her limb from limb, the fact that she was now even feeling a strange sense solidarity caused her to become even more cruel.
+Even at the best of times she would never tire of tearing this woman limb from limb,
+ but in this moment, she feels a strange sense of solidarity with her,
+ and it only makes her even crueller.
 }
 [sha:f0e1cd6af0fc56199ff99d99bed51657a53f303f]{
 -- Page 33, Offset 40341. Mods: @n.
 -- 「“私がロアを取り込んでしまえば、
-"If I take back Roa,
+"If I take back Roa―――
```

Comment:
from this point onward, the text largely resembles 37989 and on from the normal ending, so yeah, consult there.